### ISSUE TITLE
Cobertura reporter breaks on 2.6.0

### ISSUE THREAD

**Issue by vikasillumina** [tonerdo/coverlet#408]
@tonerdo I noticed similar issue reported in https://github.com/tonerdo/coverlet/issues/295
I recently upgraded coverlet version from 2.3.1 to 2.6.0 (some of my team members upgraded from 2.5.0 to 2.6.0) and our Cobertura reporter started failing.

Below is the log:
```
[Cobertura] Publishing Cobertura coverage report...

[Cobertura] No coverage results were found using the pattern '**/TestResults/coverage.*.xml' relative to '/home/ec2-user/workspace/'.  Did you enter a pattern relative to the correct directory?  Did you generate the XML report(s) for Cobertura?
```
This is our jenkin pipeline step that is failing right now with 2.6.0:
```
step([$class             : 'CoberturaPublisher',
                      autoUpdateHealth   : false,
                      autoUpdateStability: false,
                      coberturaReportFile: '**/TestResults/coverage.*.xml',
                      failUnhealthy      : false,
                      failUnstable       : false,
                      maxNumberOfBuilds  : 0,
                      onlyStable         : false,
                      sourceEncoding     : 'ASCII',
                      zoomCoverageChart  : false])
```
Same reporter works fine with 2.3.1 and my team members reported worked until 2.5.0.
Please can you look into this? If you need anything from my side please let me know.

**Comment by jpaskhay** [comment]
details of current failure specifically covered in https://github.com/tonerdo/coverlet/issues/295#issuecomment-473414110

**Comment by MarcoRossignoli** [comment]
@tonerdo you've already worked on it, do you have fast answer?I'm not so fluent on cobertura/jenkins

**Comment by tonerdo** [comment]
I believe it's because of the absence of the `source` tag. Will have to add it back

**Comment by Chocrates** [comment]
> I believe it's because of the absence of the `source` tag. Will have to add it back

@tonerdo  Any guidance on how to fix this if someone has some free time?

**Comment by MarcoRossignoli** [comment]
@Chocrates I think we need to restore this https://github.com/tonerdo/coverlet/commit/de558c5f62b46d30fb2ba15231e50fc7f41877f7#diff-a906069a8e87f7612b1e0b40fe0e38e6 to generate something similar to https://gist.github.com/apetro/fcfffb8c4cdab2c1061d#file-coverage-xml-L5 what do you think @tonerdo?

**Comment by sasivishnu** [comment]
@MarcoRossignoli I will have some time this week. I will get those old code back and see how it looks now. @tonerdo can review it after that.

**Comment by MarcoRossignoli** [comment]
@sasivishnu sure great thank's I found PR on jenkins with file sample https://github.com/jenkinsci/cobertura-plugin/commit/a024c9843ef42709058fcf4a253b9343de8ef546#diff-3977dd8133a406a118384c6775e016d2
This is the jenkins issue https://issues.jenkins-ci.org/browse/JENKINS-16252

Seem that on sources we need to add drive root i.e. C:\ on win and /... on linux and in filename the relative path.

**Comment by daveMueller** [comment]
Yes the cobertura plugin from jenkins expects a cobertura file that as a non empty source tag and relative paths in the filename attribute. This was changed to support Sourcelink in #260. In #614 I changed it back to support the jenkins plugin again and also support the Sourcelink feature. However if ```--use-source-link```-switch is true, the jenkins plugin implementation is too limited and can't display the reports. 
Nevertheless the ```ReportGenerator``` can display it ([https://github.com/danielpalme/ReportGenerator/issues/180](https://github.com/danielpalme/ReportGenerator/issues/180)).
